src/pages/journal/journal.actions.ts

```diff
@@ -12,7 +11,7 @@ export class LoadJournal implements Action {
 
 export class LoadJournalSuccess implements Action {
   readonly type = LOAD_JOURNAL_SUCCESS;
-  constructor(public payload: ExaminerWorkSchedule) {}
+  constructor(public payload: any[]) {}
```

Comment:
This should be on the list to fix once we refine the schema.

Reply:
Feel like that would be out of this refactoring's scope. Let's revisit this when most people are here in the office so that we can make a right decision.

Added a TODO for the time being

Reply:
Yeah for sure, one for later!